src/main/java/com/google/devtools/build/lib/remote/disk/DiskAndRemoteCacheClient.java

```diff
@@ -49,6 +49,11 @@ public DiskAndRemoteCacheClient(DiskCacheClient diskCache, RemoteCacheClient rem
     this.remoteCache = Preconditions.checkNotNull(remoteCache);
   }
 
+  @Override
+  public String getDisplayName() {
+    return "combined-cache";
```

Comment:
For combined cache, I would like it to first report `disk-cache` and then `remote-cache` if missed disk cache. Maybe we should move the event emission into the cache client?

Reply:
PTAL. (The architecture isn't great, but I don't see any better options.)